src/main/kotlin/fi/metatavu/metaform/server/persistence/model/MetaformReplyCreated.kt

```diff
@@ -0,0 +1,33 @@
+package fi.metatavu.metaform.server.persistence.model
+
+import java.time.OffsetDateTime
+import java.util.*
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.ManyToOne
+import javax.persistence.PrePersist
+
+@Entity
+class MetaformReplyCreated {
+
+    @Id
+    var id: UUID? = null
```

Comment:
This field does not actually exist, so use reply as Id

Reply:
Resolved